src/database/config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  development: {
+    username: process.env.DB_USER,
```

Comment:
Please move all these env variables to the envSample file as well, so people testing your branch will know the variables they will need to add in their .env file